test/README.md

```diff
@@ -0,0 +1,15 @@
+# Running tests
+
+This folder has all the system test files for lxd.
+
+## Dependencies
+
+You'll need the following dependencies installed before you can run them:
+
+    sudo apt-get install jq sqlite3 golang-go
+
+
+## Running
+
+A simple "cd test && sudo ./main.sh" should be all you need to run the system
```

Comment:
Nice, that's a good trick. How about adding a Makefile target for this to make sure it's not forgotten? something like "make test"? (I volunteer for sending a PR if that is deemed valuable).